CHANGELOG.md

```diff
@@ -705,7 +705,7 @@
 **Miscellaneous:**
 
 - Add a warning in logs saying that `type\_name` is deprecated [\#522](https://github.com/gisaia/ARLAS-server/issues/522)
-- Fix docker-compose to fit README.md [\#493](https://github.com/gisaia/ARLAS-server/issues/493) [[documentation](https://github.com/gisaia/ARLAS-server/labels/documentation)]
+- Fix docker compose to fit README.md [\#493](https://github.com/gisaia/ARLAS-server/issues/493) [[documentation](https://github.com/gisaia/ARLAS-server/labels/documentation)]
```

Comment:
It's weird to edit the changelog for that reason no ?

Reply:
Indeed! Reverted.